app/routes/public/index.js

```diff
@@ -8,13 +8,14 @@ const { set } = Ember;
 export default Route.extend({
   headData: service(),
   async model() {
-    const eventDetails = this.modelFor('public');
+    const eventDetails = this.modelFor('public', {
+      include: 'tax'
+    });
```

Comment:
Instead of this, add this in return part of async model
```js
      tax     : await eventDetails.query('tax', {}),

```

Reply:
This gives a 404 error I believe.

Reply:
Also, require the tax info in a single network request